js/web/test/training/e2e/run.js

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+const {spawn} = require('child_process');
+const startServer = require('./simple-http-server');
+const minimist = require('minimist');
+
+// copy whole folder to out-side of <ORT_ROOT>/js/ because we need to test in a folder that no `package.json` file
+// exists in its parent folder.
+// here we use <ORT_ROOT>/build/js/e2e/ for the test
+
+const TEST_E2E_SRC_FOLDER = __dirname;
+const JS_ROOT_FOLDER = path.resolve(__dirname, '../../../..');
+const TEST_E2E_RUN_FOLDER = path.resolve(JS_ROOT_FOLDER, '../build/js/e2e');
```

Comment:
suggest to use a different folder to the inference E2E test. for example, '../build/js/e2e_training'. Don't use a subfolder of it like '../build/js/e2e/training'.

Reply:
applied suggestion -- now copies to `{onnxruntime root}/build/js/e2e-training`